test/pre-formatting.test.js

```diff
@@ -26,23 +26,34 @@ process.setMaxListeners(0)
 
 describe('penthouse pre formatting tests', () => {
   it('should remove non matching media queries', done => {
+    // 1300, 1600
     const mediaToAlwaysKeep = [
       `@media all {}`,
       // going for false positives over false negatives
-      `@media oiasjdoiasd {}`
+      `@media oiasjdoiasd {}`,
+      // covering combined queries
+      `@media (min-width: 320px) and (max-width: 400px) {}`,
+      `@media (min-width: 150px) and (max-width: 1700px) {}`,
     ]
+    // 1300, 1600
     const mediaToRemoveAlways = [
       `@media print {}`,
       `@media not screen {}`,
-      `@media not (min-width: 1px) {}`
+      `@media not (min-width: 1px) {}`,
```

Comment:
minor, but to stay in-line with the code formatting in  this repo, can you remove the trailing commas, on the last array items (everywhere in this file)? 🙏

Reply:
sure, no problem